pkg/agent/kube.go

```diff
@@ -73,7 +73,8 @@ func (kube *ClusterKubeAPIClient) IsBootstrapConfigMapComplete() (bool, error) {
 	bootstrap, err := kube.Client.CoreV1().ConfigMaps("kube-system").Get(kube.ctx, "bootstrap", v1.GetOptions{})
 
 	if err != nil {
-		return false, errors.Wrap(err, "bootstrap configmap not found")
+		logrus.Tracef("bootstrap configmap not found: s%", err)
```

Comment:
Looks like the s and % need to be reversed, seeing this in unit tests:
"pkg/agent/kube.go:76:3: github.com/sirupsen/logrus.Tracef format % is missing verb at end of string"

Reply:
Doh!